Formula/scheme48.rb

```diff
@@ -21,4 +21,19 @@ def install
     system "make"
     system "make", "install"
   end
+
+  test do
+    (testpath/"hello.scm").write <<-EOS.undent
+      (display "Hello, World!") (newline)
+    EOS
+
+    expected = <<-EOS.undent
+      Hello, World!\#{Unspecific}
+
+      \#{Unspecific}
```

Comment:
What's the meaning behind this and the above `\#{Unspecific}` output? And what's going on with the many newlines?